src/__internal__/character-count/character-count.style.ts

```diff
@@ -6,14 +6,10 @@ import visuallyHidden from "../../style/utils/visually-hidden";
 const StyledCharacterCountWrapper = styled.div``;
 
 const StyledCharacterCount = styled.div<{ isOverLimit: boolean }>`
-  ::after {
-    content: " ";
-  }
-
   text-align: left;
-  font-size: 12px;
-  margin-top: 4px;
-  margin-bottom: 4px;
+  font-size: var(--fontSizes100);
```

Comment:
Yep it should have always been 14px as far as I can tell. I incorrectly believed the previous updates done to the CharacterCount was updating this as well as the text alignment so I this could be a chore as it's just updating tokens. I'll amend the commit to ensure this is included in any release notes 👍